components/influxdb/scripts/create.py

```diff
@@ -28,31 +28,50 @@ def _configure_influxdb(host, port):
     # curl -S -s "http://localhost:8086/db?u=root&p=root" '-d "{\"name\": \"cloudify\"}"  # NOQA
     import urllib
     import urllib2
+    import ast
 
-    endpoint = 'http://{0}:{1}/db'.format(host, port)
+    endpoint_for_list = 'http://{0}:{1}/db'.format(host, port)
+    endpoint_for_creation = 'http://{0}:{1}/cluster/database_configs/' \
```

Comment:
It's more common to not use backslash here but to put this in parentheses:

```python
endpoint_for_creation = (' ...... '
                                                   '.......')
```

or

```python
endpoint_for_creation = \
    ....
```